.travis.yml

```diff
@@ -2,23 +2,17 @@ language: php
 
 matrix:
     include:
-        - php: 5.3
-          dist: precise
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-        - php: 7.0
         - php: 7.1
         - php: 7.2
+        - php: 7.3
         - php: nightly
     fast_finish: true
     allow_failures:
         - php: nightly
 
 before_install:
     - phpenv config-rm xdebug.ini || true
-    - composer config --global github-oauth.github.com $GITHUB_TOKEN
```

Comment:
For whatever reason, this configuration setting is not available in the [PHP 7.3 build](https://travis-ci.org/eloquent/enumeration/jobs/457380681). It should be removed anyway, however, since the invalid token was causing Composer to [choke](https://travis-ci.org/eloquent/enumeration/jobs/457380679) every time it tried to check out a package, stalling the build for longer than necessary. 

Not sure what this was doing here in the first place, considering one does not need a token to download public packages.

Reply:
It used to be the case that Composer would hit GitHub API rate limits when running under Travis CI because of the fact that GitHub counted **every request coming from every `composer install` under Travis CI** against the same rate limit quota. The workaround was to supply a personal GitHub token.

In 2016 a change was made to GitHub's infrastructure to remove the rate limit, so it's no longer necessary: https://github.com/composer/composer/issues/4884#issuecomment-195229989

I *think* you can probably get rid of the secure environment variable in `.travis.yml` now too (it contains `GITHUB_TOKEN`).